src/containers/Auth/Logout/Logout.tsx

```diff
@@ -21,8 +23,9 @@ export const Logout: React.SFC<LogoutProps> = () => {
 
   // clear role & access permissions
   resetRole();
-
-  // TODOS: We should clear apollo cache
+  
+  // clear apollo cache
+  client.resetStore();
```

Comment:
@mdshamoon I think we should use clearStore() here as we don't want to fetch the queries again.  Check https://www.apollographql.com/docs/react/api/core/ApolloClient/#ApolloClient.resetStore

Thoughts?

Reply:
@kurund Yes, I believe clearStore() makes more sense as we don't need to refetch the queries again. Let me update the function.